runtime/compiler/ilgen/IlGenerator.cpp

```diff
@@ -988,7 +988,7 @@ TR_J9ByteCodeIlGenerator::prependEntryCode(TR::Block * firstBlock)
 
       TR::Node * firstChild = pop();
       TR::SymbolReference * monEnterSymRef =
-         isOutermostMethod() ?
+         (isOutermostMethod() || comp()->getOption(TR_EnableOSR))?
```

Comment:
Wouldn't it be better to just always use jitMethodMonitorEntry here?  If OSR isn't enabled, there is only the outermost method.

Reply:
Thanks for the suggestion. changed accordingly.